docs/output.md

```diff
@@ -546,28 +555,26 @@ For all samples:
 **Output directory: `results/Reports/[SAMPLE]/bamQC`**
 
 - `VariantCaller_[SAMPLE].bcf.tools.stats.out`
-  - RAW statistics used by MultiQC
-
-For more information about how to use Qualimap bamqc reports, see [Qualimap bamqc manual](http://qualimap.bioinfo.cipf.es/doc_html/analysis.html#id7)
+  - Raw statistics used by `MultiQC`
 
-#### MarkDuplicates reports
+For further reading and documentation see the [Qualimap bamqc manual](http://qualimap.bioinfo.cipf.es/doc_html/analysis.html#id7)
 
-[[GATK MarkDuplicatesSpark](https://software.broadinstitute.org/gatk/documentation/tooldocs/current/org_broadinstitute_hellbender_tools_spark_transforms_markduplicates_MarkDuplicatesSpark.php), Spark implementation of [Picard MarkDuplicates](https://software.broadinstitute.org/gatk/documentation/tooldocs/current/picard_sam_markduplicates_MarkDuplicates.php) locates and tags duplicate reads in a BAM or SAM file, where duplicate reads are defined as originating from a single fragment of DNA.
+#### GATK MarkDuplicates reports
 
-If the pipeline is run with the option `--no_gatk_spark` then [GATK MarkDuplicates](https://software.broadinstitute.org/gatk/documentation/tooldocs/4.1.4.0/picard_sam_markduplicates_MarkDuplicates.php) is used instead.
-
-Collecting duplicate metrics slows down performance.
-To disable them use `--skip_qc MarkDuplicates`.
+More information in the [GATK MarkDuplicates section](#gatk-markduplicates)
 
 Duplicates can arise during sample preparation _e.g._ library construction using PCR.
 Duplicate reads can also result from a single amplification cluster, incorrectly detected as multiple clusters by the optical sensor of the sequencing instrument.
 These duplication artifacts are referred to as optical duplicates.
 
+Collecting duplicate metrics slows down performance.
+To disable them use `--skip_qc MarkDuplicates`.
+
 For all samples:
 **Output directory: `results/Reports/[SAMPLE]/MarkDuplicates`**
 
 - `[SAMPLE].bam.metrics`
-  - RAW statistics used by MultiQC
+  - Raw statistics used by `MultiQC`
 
 For further reading and documentation see the [MarkDuplicates manual](https://software.broadinstitute.org/gatk/documentation/tooldocs/4.1.2.0/picard_sam_markduplicates_MarkDuplicates.php).
 
```

Comment:
#### samtools stats
[samtools stats](https://www.htslib.org/doc/samtools.html) collects statistics from `BAM` files and outputs in a text format.
Plots will show: